infra/src/main/java/com/redhat/developer/infra/CloudEventExtensions.java

```diff
@@ -0,0 +1,13 @@
+package com.redhat.developer.infra;
+
+public class CloudEventExtensions {
```

Comment:
Please rename as `BridgeCloudEventExtension`

What about implement [`CloudEventExtension`](https://github.com/cloudevents/sdk-java/blob/master/api/src/main/java/io/cloudevents/CloudEventExtension.java)? Check [this Kogito extension](https://github.com/kiegroup/kogito-runtimes/blob/main/addons/common/messaging/utils/src/main/java/org/kie/kogito/cloudevents/extension/KogitoExtension.java) as example

You can then register it globally with a startup global bean invoking [this code](https://github.com/cloudevents/sdk-java/blob/master/core/src/main/java/io/cloudevents/core/provider/ExtensionProvider.java#L65)
```java
ExtensionProvider.getInstance().registerExtension(BridgeCloudEventExtension.class, BridgeCloudEventExtension::new);
```

See [this example](https://github.com/kiegroup/kogito-runtimes/blob/0f4b959f8985c176483f10dbd8d008d849fbd522/addons/common/events/decisions/src/main/java/org/kie/kogito/eventdriven/decision/EventDrivenDecisionController.java#L108) about how to extract an extension from a message and [this other example](https://github.com/kiegroup/kogito-runtimes/blob/0f4b959f8985c176483f10dbd8d008d849fbd522/addons/common/messaging/utils/src/main/java/org/kie/kogito/cloudevents/CloudEventUtils.java#L59-L84) about how to serialise a message + extension

It probably introduces some overhead for now but it centralises all extension in a single way so it should be easier to add/handle them
Wdyt?

Reply:
Thanks for the links. 

Yep I think it's useful to centralise this stuff now. I think we will very likely be adding additional extensions in the future.